lib/bundler/lazy_specification.rb

```diff
@@ -29,7 +29,7 @@ def initialize(name, version, platform, source = nil)
       @name          = name
       @version       = version
       @dependencies  = []
-      @platform      = platform
+      @platform      = platform || Gem::Platform::RUBY
```

Comment:
I don't have any context in this codebase, so I have no idea -- is this a safe default assumption to make?